### PR TITLE
Added front door for reform scan

### PR DESCRIPTION
**Pull request by nitinprabhu**

### CODE REVIEW

File: frontdoor.tf [hunk 1]
```diff
@@ -0,0 +1,63 @@
+resource "azurerm_frontdoor" "example" {
+  name                                         = "reform-scan-front-door-${var.env}"
```
Comment (doncem) [inline frontdoor.tf:2]:
```suggestion
  name                                         = "${var.product}-front-door-${var.env}"
```
?

File: frontdoor.tf [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      http_port   = 80
+      https_port  = 443
+    }
+
+    load_balancing_name = "loadbalance-setting"
+    health_probe_name   = "health-probe-setting"
+  }
+
+  frontend_endpoint {
+    name                                    = "reform-scan-appgw-endpoint"
```
Comment (doncem) [inline frontdoor.tf:39]:
```suggestion
    name                                    = "${var.product}-appgw-endpoint"
```

**Review by doncem — COMMENTED** [review]
only commenting on what i can. otherwise i'm blindfolded

**Comment by nitinprabhu** [comment]
> only commenting on what i can. otherwise i'm blindfolded

will implement the suggestions once we have the whole thing sort of working :) 
There are other bits which needs to be done for it to be working

**Review by jrjohnson87 — APPROVED** [review]

**Review by gunnertwin — APPROVED** [review]

**Pull request closed without merging**